content/en/user-guide/aws/stepfunctions/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Step Functions"
+linkTitle: "Step Functions"
+categories: ["LocalStack Community"]
+description: Get started with Amazon Web Services Step Functions
+---
+
+Amazon Web Services (AWS) Step Functions is a serverless workflow engine that integrates with AWS Lambda and other AWS services. It allows us to define workflows using Amazon States Language, a JSON-based structured language, to provide the necessary configurations and run the workflow. Step Functions allow you to define standard workflows and express workflows for long-running workflows and high-volume event processing, respectively.
+
+LocalStack provides Step Functions support via the Community offering. You can use the Step Functions API to create, execute, update, and delete state machines. The supported APIs are available over our [feature coverage page]({{< ref "feature-coverage" >}}).
+
+## Getting started
+
+In this getting started guide, you'll learn how to make a basic usage of Step Functions over LocalStack. This guide is intended for users who wish to get more acquainted with Step Functions, and assumes you have basic knowledge of the AWS CLI (and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script). First, start your LocalStack instance using your preferred method, then run the following commands:
+
+1. Create a State Machine using the `create-state-machine` command and specify the name of the state machine, the Amazon States Language definition, and the role ARN to use for the state machine:
+   {{< command >}}
+   $ awslocal stepfunctions create-state-machine \
+        --name "WaitExecution" \
+        --definition '{
+        "StartAt": "WaitExecution",
+        "States": {
+            "WaitExecution": {
+            "Type": "Wait",
+            "Seconds": 10,
+            "End": true
+            }
+        }
+        }' \
+        --role-arn "arn:aws:iam::000000000000:role/stepfunctions-role"
+   {{< /command >}}
+   In the above example, we have created a state machine named `WaitExecution`, with a single state, `WaitExecution`, which waits for 10 seconds before completing the execution. The role ARN is used to grant the state machine access to other AWS services, and you can create an IAM role within LocalStack as well. The output of the above command is the ARN of the state machine:
```

Comment:
Using the same string for the state machine as for the state seems confusing. I'd suggest at least changing the state machine name to something else, e.g. `WaitMachine`.